curriculum/challenges/english/01-responsive-web-design/basic-css/use-an-id-attribute-to-style-an-element.english.md

```diff
@@ -12,6 +12,8 @@ However, an <code>id</code> is not reusable and should only be applied to one el
 Here's an example of how you can take your element with the <code>id</code> attribute of <code>cat-photo-element</code> and give it the background color of green. In your <code>style</code> element:
 <blockquote>#cat-photo-element {<br>&nbsp;&nbsp;background-color: green;<br>}</blockquote>
 Note that inside your <code>style</code> element, you always reference classes by putting a <code>.</code> in front of their names. You always reference ids by putting a <code>#</code> in front of their names.
+  
+One thing to note is that using ID selectors to style elements should be avoided. Using IDs for styling individual elements loses out on much of the functionality of CSS sheets, and because they have such high specificity, they override most other selectors for styling.
```

Comment:
Can we be a bit more concise here? We only need to mention why IDs should be avoided.
Something like:

>**Note:**
ID selectors to style elements should be avoided. Due to their high specificity, they can override class and tag selectors.

Reply:
Sure, sounds good.